docs/source/_static/main.css

```diff
@@ -0,0 +1,3 @@
+col {
+    width: 50% !important;
+}
```

Comment:
This file is needed as there is `colgroup` html tag which apply width 10% to the right and 90% to the left.
Not sure where does first appear, can't inspect that in pytorch docs.
So manually setting it.